client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -1238,7 +1241,6 @@ public void onSuccess(ByteBuffer response) {
                 groupedBatchId,
                 Arrays.toString(batchIds));
             pushState.removeBatch(groupedBatchId, hostPort);
-            // TODO Need to adjust maxReqsInFlight if server response is congested, see CELEBORN-62
```

Comment:
emm... we remove those TODO? were they implemented?

Reply:
yes, it has been done in the wrappedCallback